### ISSUE TITLE
Notify the user he/she can edit his/her responses.

### ISSUE THREAD

**Issue by illegalprime** [HackGT/registration#8]
Whether by email or something else, or on the successful submission page, provide a link the user can visit to change his responses. The actual mechanics of editing responses is a separate issue described here: #7 .

This should be completed after #7

**Comment by petschekr** [comment]
Closed by #15
